hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson

```diff
@@ -40,6 +40,7 @@
              "{proj_root}/hw/ip/spi_device/dv/spi_device_sim_cfg.hjson",
              "{proj_root}/hw/ip/sram_ctrl/dv/sram_ctrl_main_sim_cfg.hjson",
              "{proj_root}/hw/ip/sram_ctrl/dv/sram_ctrl_ret_sim_cfg.hjson",
+             "{proj_root}/dv/sv/tl_agent/dv/tl_agent_sim_cfg.hjson",
```

Comment:
Could you add a separate heading `// Unit tests for UVCs. ` at the start and move this there? This section is for all IPs.